distribution/lib/Standard/Base/0.0.0-dev/src/Data/XML.enso

```diff
@@ -136,6 +137,7 @@ type XML_Document
     name self = self.root_element.name
 
     ## GROUP Selections
+       ICON parse3
```

Comment:
Why is it called parse3? Is there a parse 1 and 2?

Reply:
That was the one that felt best in the figma file. Am sure we will want to tune and replace over time.